fitbert/fitb.py

```diff
@@ -12,13 +12,15 @@
 class FitBert:
     def __init__(
         self,
+        device=None,
         model=None,
         tokenizer=None,
         model_name="bert-large-uncased",
         mask_token="***mask***",
     ):
         self.mask_token = mask_token
         self.delemmatizer = Delemmatizer()
+        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
I think we should check if the user passed in `device` here: if they did, it's probably them trying to tell us to use GPU or CPU. Maybe we should check if they pass `device="cuda"` or `device="cpu"`, and respect that. If neither, check if it is truthy, in which case set `device="cuda"`, and if falsey, set `device="cpu"`?

Reply:
Agreed, we can have such fix here. Although I would wait till we finalize 'insides' first, then add options etc.